.github/workflows/scripts/pr-gh-project.js

```diff
@@ -334,7 +334,7 @@ async function processOpenOrEditAction() {
     console.log('No milestones on issue(s) for this PR');
   } else if (keys.length > 1) {
     console.log('More than one milestone on issues for this PR');
-  } else {
+  } else if (event.pull_request.user.type !== 'Bot') {
```

Comment:
I'm honestly unsure if a type of `Bot` is accurate. The github api docs don't make the acceptable values entirely clear, but I found some comments on stack overflow that suggest `Bot` is correct. 

In the worst case scenario, `pr-gh-project` will still attempt to update the milestone for backport PRs raised by the backport bot.

Reply:
Alternatively, we can attempt to strip any existing closing keywords from the body of the original PR when raising the backport.

Reply:
Is there anymore inclusive checks we could do? Like lowercase or check for includes 'bot' to increase the likelihood of this being correct?